update_command.php

```diff
@@ -94,34 +94,41 @@
         } else {
             // get all branches
             $allBranches = explode("\n", cmd('git branch -r', $MODULE_DIR));
+            $allBranches = array_filter($allBranches, fn($branch) => !str_contains($branch, 'HEAD ->'));
```

Comment:
What does this line do? It's not immediately clear and doesn't seem to be covered by the comment you made about the `.github` master branch.

Reply:
It filters the array of string branch names and removes anything that contains the string `HEAD ->`